src/core_landice/Registry.xml

```diff
@@ -1033,29 +1090,54 @@ is the value of that variable from the *previous* time level!
                 />
-                <var name="ismip6shelfMelt_zOcean" type="real" dimensions="nISMIP6OceanLayers" units="m" packages="ismip6Melt"
+                <var name="ismip6shelfMelt_zOcean" type="real" dimensions="nISMIP6OceanLayers" units="m" packages="ismip6ShelfMelt"
                      description="depth coordinate associated with ismip6shelfMelt_3dThermalForcing field, input to model"
                 />
-                <var name="ismip6shelfMelt_offset" type="real" dimensions="nCells" units="kg m^{-2} s^{-1}" packages="ismip6Melt" default_value="0.0"
+                <var name="ismip6shelfMelt_offset" type="real" dimensions="nCells" units="kg m^{-2} s^{-1}" packages="ismip6ShelfMelt" default_value="0.0"
                      description="Optional field of offset that should be applied to the melt field calculated by the ISMIP6 melt method.  Can be used to run in anomaly mode, but this is not the anomaly.  See Registry.xml for details."
-                        />
+                />
+                <!-- Variables needed for ISMIP6 grounded face melt parameterization -->
+                <var name="groundedMarineMarginMask" type="integer" dimensions="nCells Time" units="none" time_levs="1" packages="ismip6GroundedFaceMelt"
+                     description="mask of grid cells that are grounded and adjacent to the ocean. 0=interior, 1=grounded ice margin"
+                />
+                <var name="faceMeltRateApplied" type="real" dimensions="nCells Time" units="m^{3} s^{-1}" packages="ismip6GroundedFaceMelt"
+                     description="Volumetric submarine melt rate applied at grounded marine margin"
+                />
+		<var name="faceMeltSpeed" type="real" dimensions="nCells Time" units="m s^{-1}" packages="ismip6GroundedFaceMelt"
+		     description="linear speed of submarine melting at grounded glacier front"
+  		/>
+                <var name="subglacialWaterFlux" type="real" dimensions="nCells Time" units="m day^{-1}" packages="ismip6GroundedFaceMelt" default_value="0.0"
+                     description="Catchment-wide subglacial runoff normalized by modern submerged face area, in units used by ismip6 grounded marine melt parameterization, with zeros where ISMIP6 forcing file has negative runoff"
+                />
+                <var name="ismip6Runoff" type="real" dimensions="nCells Time" units="kg m^{-2} s^{-1}" packages="ismip6GroundedFaceMelt" default_value="0.0"
+                     description="Runoff forcing for ISMIP6 grounded ice melting method, input to model" 
+                />
+		<var name="ismip6_2dThermalForcing" type="real" dimensions="nCells Time" units="deg. C" packages="ismip6GroundedFaceMelt" default_value="2.0"
+	 	     description="thermal forcing for ISMIP6 grounded marine melt method, input to model"
+		/>
+		<var name="meltingThickness" type="real" dimensions="nCells Time" units="m" time_levs="1"
```

Comment:
I suggest renaming this facemeltingThickness or something to avoid confusion with surface melt, in the SMB sense.